go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/google/docsy
+
+go 1.12
```

Comment:
Is this the minimum required version of Go or a precise version constraint -- i.e., v1.12 must be used?

Reply:
This is the minimum required version of Go. Higher versions of Go are fine.